### PR TITLE
display application metadata in instance detail page

### PR DESCRIPTION
**Pull request by tetsushiawano**
This PR adds the application metadata in the instance detail page. Solve the issue of #642 
I created this PR without the preferences from the maintainers so I would like your feedback.
Thank you in advance.

Desktop View

Smartphone View

### CODE REVIEW

**Comment by tetsushiawano** [comment]
I run Travis CI with master branch and result was same. 
https://travis-ci.org/tetsushiawano/spring-boot-admin/jobs/340133403

**Comment by joshiste** [comment]
> Travis CI is failing with the modified branch.

yeah it's du to changes in the snapshots for spring cloud. I've pushed the changes to master. If you rebase the build should pass.

**Comment by tetsushiawano** [comment]
I changed metadata to computed. Also, rebase the upstream/master to the modified branch.

Let me paste screenshot after the changes.

PC View


Smartphone View


Thank you for your support.

File: spring-boot-admin-server-ui/src/main/frontend/views/instances/details/details-metadata.vue [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    data: () => ({
+      metadata: {}
+    }),
+    computed: {
+      isEmptyMetadata() {
+        return Object.keys(this.metadata).length <= 0;
+      }
+    },
+    created() {
+      this.metadata = this.instance.registration.metadata;
```
[the remark this answers is not in the document]
Comment (tetsushiawano) [reply spring-boot-admin-server-ui/src/main/frontend/views/instances/details/details-metadata.vue]: I change to computed with the following commit. e248feded985de1fdc000c9866890d54c4bea340

**Comment by joshiste** [comment]
merged with 1770b4c0a58ddf4761b7341dd9e1748794914306

**Pull request closed without merging**